app/views/confirm.scala.html

```diff
@@ -14,21 +14,17 @@
  * limitations under the License.
  *@
 
-@import _root_.template.Helper._
-@import helpers._
 @import models.pages.Summary
 @import play.api.data._
 @import uk.gov.hmrc.govukfrontend.views.html.components._
-@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
 @import uk.gov.hmrc.govukfrontend.views.html.helpers.formWithCSRF
-@import template.FormatRefNum
 
 @this(  layout: pageLayout,
         govukRadios: GovukRadios,
         govukButton: GovukButton,
         govukErrorSummary: GovukErrorSummary,
         govukCharacterCount: GovukCharacterCount,
-        formWithCSFR: formWithCSRF)
+        formWithCSRF: formWithCSRF)
```

Comment:
Is this variable actually used - I would expect the renaming to be done somewhere else